db/payment_icons.yml

```diff
@@ -1486,6 +1486,10 @@
   name: cebuana
   label: Cebuana
   group: other
+-
+  name: futurepay
```

Comment:
I will also recommend changing the name (and logo file name) to `futurepay_mytab` so that it aligns with other naming convention.

Reply:
Sure, I will do it